__tests__/app.test.js

```diff
@@ -90,6 +90,7 @@ describe("GET /api/reviews/:review_id", () => {
           category: expect.any(String),
           owner: expect.any(String),
           created_at: expect.any(String),
+          comment_count: 0,
```

Comment:
We shouldn't ever amend existing tests when new functionality is added. Rather this should be a new test, only checking for comment_count and the previous tests if they are robust shouldn't break